README.md

```diff
@@ -300,6 +300,7 @@ A curated list of [awesome](https://github.com/sindresorhus/awesome) Open Policy
 - [gator CLI](https://open-policy-agent.github.io/gatekeeper/website/docs/gator/) - Command line unit test runner for OPA Gatekeeper
 - [ocov](https://github.com/C5T/ocov) - Colors `opa test --coverage` reports in the terminal
 - [opa-codecov](https://github.com/SVilgelm/opa-codecov) - Convert OPA test coverage report to a JSON format supported by Codecov
+- [GitHub Actions for OPA Rego Policy Test](https://github.com/masterpointio/github-action-opa-rego-test) - GitHub Action to automate testing for your OPA Rego policies and generates a report.
```

Comment:
Can we use the title of the project for the text here instead? I.e:

```
[github-action-opa-rego-test](https://github.com/masterpointio/github-action-opa-rego-test) - GitHub Action to automate testing for your OPA Rego policies and generate a report.
```

It feels like the text is mostly repeated in the description later anyway.

Reply:
I also made a PR over here https://github.com/open-policy-agent/opa/pull/6954
I believe you're a maintainer there as well? Would be nice to get it added there too! Thanks!